components/message-card.tsx

```diff
@@ -0,0 +1,112 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { FaArrowUp, FaCalendar, FaPhone } from 'react-icons/fa6';
+import { MdEmail } from 'react-icons/md';
+
+import { cn } from '~/lib/utils';
+
+import { Button } from './ui';
+
+export default function messageCard({
+  className,
+  image,
+  name,
+  quote,
+  quoteBelow,
+  readMorePath,
+  locale,
+  more,
+  details,
+}: {
+  className?: string;
+  image: string;
+  name: string;
+  quote: string;
+  quoteBelow?: string;
+  readMorePath?: string;
+  more?: string;
+  locale: string;
+  details?: {
+    email: string;
+    phone: string;
+    session: string;
+  };
+}) {
+  return (
+    <article
+      className={cn(
+        'grid gap-5',
+        'grid-cols-[5rem,auto] lg:grid-cols-[min(30%,24rem),auto]',
+        'p-2 sm:p-2 md:p-6 xl:p-8',
+        'rounded-xl border border-primary-700 bg-neutral-50',
+        className
+      )}
+    >
+      <Image
+        src={image}
+        alt={name}
+        height={500}
+        width={500}
+        className="aspect-[5/6] rounded-xl object-cover max-lg:w-20 lg:row-span-2"
+      />
+      <h3 className="mb-0 h-fit self-center max-sm:text-lg lg:text-4xl">
+        {name}
+      </h3>
+      <p
+        className={cn(
+          'max-lg:col-span-2 lg:text-xl',
+          details && 'lg:row-span-2'
+        )}
+      >
+        {quote}
+        &nbsp;
+        {readMorePath && (
+          <Button
+            asChild
+            className="inline-flex items-center gap-1"
+            variant="link"
+          >
+            <Link href={`/${locale}${readMorePath}`}>
+              {more}
+              <span className="rotate-90">
+                <FaArrowUp
+                  className={cn(
+                    'mx-auto animate-bounce',
+                    'size-2 md:size-3 lg:size-4'
+                  )}
+                />
+              </span>
+            </Link>
+          </Button>
+        )}
+        {quoteBelow ? (
+          <>
+            <br />
+            <br />
+            {quoteBelow}
+          </>
+        ) : (
+          ''
+        )}
+      </p>
+      {details && (
+        <footer className="max-lg:col-span-2">
+          <ul>
+            <li className="flex items-center gap-2">
+              <MdEmail className="fill-primary-700" />
+              {details?.email}
+            </li>
+            <li className="flex items-center gap-2">
+              <FaPhone className="fill-primary-700" />
+              {details?.phone}
+            </li>
+            <li className="flex items-center gap-2">
+              <FaCalendar className="fill-primary-700" />
+              {details?.phone}
```

Comment:
We won't have `?` checks since we've already checked for `details` being a truthy value up above.